konfera/event/views.py

```diff
@@ -127,3 +128,13 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
```

Comment:
I think it is because of what he said in an earlier comment in #179, that an order can have more than one ticket if added through the admin panel, so it is a one-to-many relationship and can have tickets from different events, so I think the info about the event only need to be shown in the ticket. And the slug really needs to be removed.